client/mm/exchange_adaptor.go

```diff
@@ -1545,11 +1545,11 @@ func cexTradeBalanceEffects(trade *libxc.Trade) (effects *BalanceEffects) {
 
 	if trade.Sell {
 		effects.Settled[trade.BaseID] = -int64(trade.Qty)
-		effects.Locked[trade.BaseID] = trade.Qty - trade.BaseFilled
+		effects.Locked[trade.BaseID] = utils.SafeSub(trade.Qty, trade.BaseFilled)
```

Comment:
This requires comment too. We can't brush math inconsistencies under the rug.

Reply:
I've added a log. The underlying issue in the coinbase implementation was fixed.